addons/web/static/src/js/widgets/debug_manager.js

```diff
@@ -368,13 +368,26 @@ DebugManager.include({
             metadata.create_date = field_utils.format.datetime(createDate);
             var modificationDate = field_utils.parse.datetime(metadata.write_date);
             metadata.write_date = field_utils.format.datetime(modificationDate);
-            new Dialog(this, {
+            var dialog = new Dialog(this, {
                 title: _.str.sprintf(_t("Metadata (%s)"), self._action.res_model),
                 size: 'medium',
                 $content: QWeb.render('WebClient.DebugViewLog', {
                     perm : metadata,
                 })
-            }).open();
+            });
+            dialog.open().opened(function () {
+                dialog.$el.on('click', 'a[data-action="set_noupdate"]', function (ev) {
```

Comment:
as far as i can tell, it does not set no update, it toggles the flag. So, you should use toggle_noupdate instead of set_noupdate